storage/log_storage.go

```diff
@@ -156,7 +156,8 @@ type CountByLogID map[int64]int64
 
 // LogMetadata provides access to information about the logs in storage
 type LogMetadata interface {
-	// GetActiveLogs returns a list of the IDs of all the logs that are configured in storage
+	// GetActiveLogIDs returns a list of the IDs of all the logs that are
+	// configured in storage and are in the ACTIVE or DRAINING state.
```

Comment:
Maybe change this to something like "eligible to have entries sequenced" so it won't get out of date.

Reply:
LGTM apart from that.

Reply:
Done.